arm-analysisservices/2016-05-16/swagger/analysisservices.json

```diff
@@ -525,6 +528,28 @@
           "description": "An array of administrator user identities."
         }
       }
+    },
+    "BackupConfiguration": {
+      "description": "An object that represents backup configurations",
+      "type": "object",
+      "properties": {
+        "storageAccount": {
+          "type": "string",
+          "description": "The name of storage account for backup configuration. It is in the format of 'Resource Group Name/Storage Account Name'"
```

Comment:
@johanste I synced with the one who implemented this backup functionality, we have a plan to change to storage account resource id in the near future. We will come back and change here then. For now it is inline with what we have for ARM.